src/main/java/com/training/web/pages/flipkart/FlipkartHomePage.java

```diff
@@ -1,14 +1,24 @@
 package com.training.web.pages.flipkart;
 
+import com.codeborne.selenide.SelenideElement;
 import com.training.basepages.FlipkartBasePage;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import static com.codeborne.selenide.Condition.enabled;
 import static com.codeborne.selenide.Condition.visible;
 import static com.codeborne.selenide.Selenide.$x;
 
 public class FlipkartHomePage extends FlipkartBasePage {
-
+    private final Logger logger = LoggerFactory.getLogger(this.getClass());
     private String searchBox = "//input[contains(@title,'Search for products')]";
     private String popup = "//button[@class='_2KpZ6l _2doB4z']";
-    private String submit ="//button[@class='L0Z3Pu']";
+    private String submit = "//button[@class='L0Z3Pu']";
+    private String facebookLink = "//a[text()='Facebook']";
+    private String twitterLink = "//a[text()='Twitter']";
+    private String youtubeLink = "//a[text()='YouTube']";
```

Comment:
the pattern of this xpath looks same - so can we rather a single xpath and do a string.format where it is used?